geode-core/src/main/java/org/apache/geode/management/DistributedRegionMXBean.java

```diff
@@ -290,8 +290,16 @@
   boolean isPersistentEnabled();
 
   /**
-   * Returns the aggregate entry size (in megabytes) of all entries. This will provide a correct
-   * value only if the eviction algorithm has been set to {@link EvictionAlgorithm#LRU_MEMORY}.
+   * Returns the aggregate entry size (in bytes) of all entries. 
+   * For replicated regions, provides a value only if the eviction algorithm
```

Comment:
Looks good! If you want a paragraph then use `<p>` by itself (JDK javadocs are like this):
```
* Returns the aggregate entry size (in bytes) of all entries. 
*
* <p>
* For replicated regions, provides a value only if the eviction algorithm
```
I realize most of the JMX javadocs are missing this, and I'm not requesting changes -- just commenting.